README.md

```diff
@@ -23,6 +23,7 @@ A curated list of awesome AWK frameworks, libraries, software and resources.
  * [Awk Introduction and Resources by Frank Moraes](https://www.whoishostingthis.com/resources/awk/)
  * [A guide to intermediate awk scripting](https://opensource.com/article/19/11/intermediate-awk-scripting)
  * [Advance your awk skills with two easy tutorials](https://opensource.com/article/19/10/advanced-awk)
+ * [awk is the coolest tool you don't know](gemini://drewdevault.com/2021/05/03/awk-is-the-coolest-tool-you-dont-know.gmi)
```

Comment:
Thank you for your contribution!

Would you mind changing link to Gemini-HTTP portal instead of plain Gemini URL (for instance https://portal.mozz.us/gemini/drewdevault.com/2021/05/03/awk-is-the-coolest-tool-you-dont-know.gmi)?

I'd like to not force people getting Gemini client as vanilla Firefox / Chrome does not open the gemini link out-of-the box.

Reply:
I think the author has made that article only available on gemini.

I tried searching for it on http but couldn't find it.

Reply:
Let me know if you have better luck finding it

Reply:
Do you still want to add it even if it is only on gemini?

Reply:
I understand the article's author published on gemini only, but thanks to [Gemini-to-HTTP portals](https://en.wikipedia.org/wiki/Gemini_(protocol)#Gemini_clients) anyone can see it even with vanilla browser.

So I'd like to ask you to change the link to https://portal.mozz.us/gemini/drewdevault.com/2021/05/03/awk-is-the-coolest-tool-you-dont-know.gmi or alternative one.